app/posts/apply-for-teacher-training/2020-02-11-training-with-a-disability-iteration.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Giving details about disability (iteration)
+description:
+date: 2020-02-11
+---
+This design is [an iteration of the question as designed for the pilot](/apply-for-teacher-training/training-with-a-disability). This question was removed prior to launch due to a lack of confidence in how we were asking a sensitive and possibly contentious question.
+
+## User needs
+
+As a candidate
+: I need to inform a provider of a disability, health issue or access need I have,
```

Comment:
@fofr and I have been avoid this syntax. Suggest just losing the `:` but still having separate lines.

Reply:
Any reason why? Semantics?